embassy-stm32/src/rcc/g4.rs

```diff
@@ -229,16 +229,15 @@ pub(crate) unsafe fn init(config: Config) {
         _ => unreachable!(),
     };
 
-    assert!(max::SYSCLK.contains(&sys));
+    rcc_assert!(max::SYSCLK.contains(&sys));
 
     // Calculate the AHB frequency (HCLK), among other things so we can calculate the correct flash read latency.
     let hclk = sys / config.ahb_pre;
-    assert!(max::HCLK.contains(&hclk));
+    rcc_assert!(max::HCLK.contains(&hclk));
 
     let (pclk1, pclk1_tim) = super::util::calc_pclk(hclk, config.apb1_pre);
     let (pclk2, pclk2_tim) = super::util::calc_pclk(hclk, config.apb2_pre);
-    assert!(max::PCLK.contains(&pclk2));
```

Comment:
Was this meant to be removed, or was it a typo and should have been kept and changed to &pclk1?

Reply:
I think it was a typo, so I removed it because it was duplicated. If needed, I can create another ticket to fix it.